Player.js

```diff
@@ -0,0 +1,18 @@
+const PLAYER_VERSION = '0.1';
+
+class Player {
+  static getVersion() {
```

Comment:
I'm not too familiar with ES6, isn't it possible to `static const version = '0.1'` in the Player class?